draft-irtf-cfrg-hpke.md

```diff
@@ -951,6 +989,26 @@ For X25519 and X448, public keys and Diffie-Hellman outputs MUST be validated
 as described in {{RFC7748}}. In particular, recipients MUST check whether
 the Diffie-Hellman shared secret is the all-zero value and abort if so.
 
+### Future KEMs {#future-kems}
+
+{{kem-security}} lists security requirements on a KEM used within HPKE.
+
+The `AuthEncap` and `AuthDecap` functions are OPTIONAL. If a KEM algorithm
+does not provide them, only the Base and PSK modes of HPKE are supported.
+Future specifications which define new KEMs MUST indicate whether or not
+these modes are supported.
+
+A KEM algorithm MAY define multiple different equivalent encoding
+algorithms for KEM public keys or KEM ciphertexts, with different output
+lengths. In this case `Npk` and `Nenc` MUST indicate the maximum output
+length. It is left to the application using HPKE to define how entities
+communicate which encoding is used.
+
+The notation `pk(skX)`, depending on its use and the KEM, is either the
```

Comment:
This feels out of place in this section. Perhaps this should go underneath the KEM interface description? (That is, right below the list of functions.)